### ISSUE TITLE
Building Graphql explorer fails

### ISSUE THREAD

**Issue by deanmarcussen** [OrchardCMS/OrchardCore#4584]
We've had a few updates to packages recently, but I think we have not been able to build the graphql explorer properly with the new updates.

The webpack script `npm run-script build` fails for me with a lot of loader / module parsing issues

![build-graphql](https://user-images.githubusercontent.com/13782679/67156805-2613a900-f31b-11e9-9d48-d63ffa84e03b.PNG)

/cc @jrestall

**Comment by jrestall** [comment]
This is related to https://github.com/graphql/graphiql/issues/617#issuecomment-539034320

Suggested temporary fix until the upstream fixes are made is:
```
{
  test : /\.(ts|ts.map|js.map)$/,
  include : [".node_modules/graphql-language-service-interface"],
  use : "ignore-loader"
}
```

**Comment by jrestall** [comment]
I've fixed the warnings in the build. The package has been getting built correctly since they aren't errors though.

**Comment by deanmarcussen** [comment]
Thanks @jrestall I am unfamiliar with it, so wasn't sure if it was building correctly or not

**Comment by acao** [comment]
@jrestall this issue has been fixed in 0.17.0, had to do with performing dynamic requires without specifying extensions (GraphQLLanguageService class). please see the webpack example for details, incl the babel config. ping me in discord if you have any issues

also, enjoy the esm modules :D

**Comment by jrestall** [comment]
Thanks @acao, the webpack build is running a lot cleaner now :)
